### PR TITLE
Voter, Election schema with some endpoints

### PR DESCRIPTION
**Pull request by eashwarmp**
I've created the Voter and Elector schema. Have also added some endpoints(getSecretBallot to get voter secret location) and createElection to get the election created

### CODE REVIEW

File: server/data/voters.js [hunk 1]
```diff
@@ -1,22 +1,44 @@
...
     {
         "id": 1,
-        "voterName": "Do something nice for someone I care about",
+        "voterName": "Check something 1",
         "hasVoted": false,
+        "electionId": 123,
+        "secretBallot" : "eeeaaa",
+        "hasRegistered": true,
+        "isElectionOwner": true,
         "priority": "Important",
```
Comment (meyeralex31) [inline server/data/voters.js:10]: what is this?

File: server/models/voterModel.js [hunk 2]
```diff
@@ -9,12 +12,32 @@ const voterSchema = mongoose.Schema (
         profileId : {
             type: String,
             required: true
-        }
+        },
+        electionId: {
+            type: String,
+            // required: true
+        },
+        secretBallot : {
```
Comment (meyeralex31) [inline server/models/voterModel.js:20]: is this the secret location?

**Comment by meyeralex31** [comment]
Can we have the ids auto generate? Versus front end worrying about creating them?

File: server/data/voters.js (hunk 1 above)
Comment (eashwarmp) [reply server/data/voters.js:10]: This is just a sample json to put into DB. Just to make sure it's getting inserted as expected

File: server/models/voterModel.js (hunk 2 above)
Comment (eashwarmp) [reply server/models/voterModel.js:20]: Yes that's right

**Comment by eashwarmp** [comment]
Sure I can make the id auto generate from BE

File: server/models/electionModel.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            question : {
+                type: String,
+                required: true
+            },
+            options: [ {
+                option : {
+                    type: String,
+                    required: true
+                },
+            questionId: mongoose.Schema.Types.ObjectId,
```
Comment (meyeralex31) [inline server/models/electionModel.js]: this is in the wrong place. This needs to be moved up.

File: server/models/electionModel.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+const electionSchema = mongoose.Schema (
+    {   
+        questions: [ {
+            question : {
+                type: String,
+                required: true
+            },
+            options: [ {
+                option : {
```
Comment (meyeralex31) [inline server/models/electionModel.js:24]: does this need to be an array of objects or can it just be an array of strings

File: server/models/electionModel.js (hunk 3 above)
Comment (eashwarmp) [reply server/models/electionModel.js]: Have changed it now with the latest commit

**Pull request closed without merging**